app/models/game.rb

```diff
@@ -0,0 +1,9 @@
+class Game < ActiveRecord::Base
+  has_many :memberships
+  has_many :users, through: :memberships
+
+  def <<(user)
```

Comment:
The purpose of the game object is to associate users together. I want to avoid game from becoming a god object, so I think its most array-like feature will always be adding users. I could alias `#<<` to `#add_user` and if things become unclear in the future change the method name entirely.